### ISSUE TITLE
How to disable checkpoints?

### ISSUE THREAD

**Issue by GospodinNoob** [tensorflow/tensorflow#10394]
This is a classifier:
classifier = SKCompat(tf.contrib.learn.DNNClassifier(feature_columns=feature_columns, 
                                            hidden_units=[7, 4, 2], 
                                            n_classes=2,
                                            optimizer=tf.train.GradientDescentOptimizer(0.08)
                                                    )
                     )

and when i try to fit model, i get this:
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Saving checkpoints for 1 into C:\Users\home\AppData\Local\Temp\tmprit6vryq\model.ckpt. 
INFO:tensorflow:loss = 0.71007, step = 1
INFO:tensorflow:Loss for final step: 0.71007.
WARNING:tensorflow:float64 is not supported by many models, consider casting to float32.
INFO:tensorflow:Restoring parameters from C:\Users\home\AppData\Local\Temp\tmprit6vryq\model.ckpt-1

How to disable saving? It takes 3-4 seconds. It's too long for me. Thanks for helping!

I want to fit my model "online" (1 example, 1 step).

**Comment by jart** [comment]
Please don't crosspost [StackOverflow questions](https://stackoverflow.com/questions/44330025/how-to-disable-checkpoints-in-fit-tensorflow). CC: @alextp
